waffle-doppelganger/src/index.ts

```diff
@@ -0,0 +1,46 @@
+import {utils, Contract, Wallet, ContractFactory} from 'ethers';
+import {FunctionDescription} from 'ethers/utils/interface';
+import {EventFragment, FunctionFragment, ParamType} from 'ethers/utils/abi-coder';
+import DoppelgangerContract from './Doppelganger.json';
+
+type ABI = Array<EventFragment | FunctionFragment | ParamType> | string;
```

Comment:
Hmm, I think we already have ABI type somewhere in Waffle

Reply:
You should be able to get this type from ethers?